src/api/app/views/webui/users/tokens/new.html.haml

```diff
@@ -51,7 +51,7 @@
 - content_for :ready_function do
   :plain
     // Default type value: 'runservice'
-    $('#token_type_runservice').attr('checked', true);
+    $('#token_type_service').attr('checked', true);
```

Comment:
Why is this done in JS anyway? This is just an HTML attribute you can set in the `collection_radio_buttons` loop on line 16 no?

I think you also want to include `include_hidden: false` there too.

https://api.rubyonrails.org/classes/ActionView/Helpers/FormOptionsHelper.html#method-i-collection_radio_buttons-label-Gotcha

Reply:
Done. Thanks!